app/services/set_order_items.rb

```diff
@@ -0,0 +1,25 @@
+class SetOrderItems
+  SHIPPING_COSTS = 100
```

Comment:
```suggestion
  SHIPPING_COSTS = 100
  
```

espaço entre declações para facilitar a leitura